lib/core.ps1

```diff
@@ -318,6 +318,39 @@ function Get-HelperPath {
     }
 }
 
+function Get-CommandPath {
+    [CmdletBinding()]
+    [OutputType([String])]
+    param(
+        [Parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [String]
+        $Command
+    )
+
+    begin {
+        $userShims = Convert-Path (shimdir $false)
+        $globalShims = fullpath (shimdir $true) # don't resolve: may not exist
+    }
+
+    process {
+        try {
+            $comm = Get-Command $Command -ErrorAction Stop
+        } catch {
+            return $null
+        }
+        $commandPath = if ($comm.Path -like "$userShims*" -or $comm.Path -like "$globalShims*") {
+            Get-ShimTarget ($comm.Path -replace '\.exe$', '.shim')
```

Comment:
If a bin includes both `*.ps1` and `*.exe` shims (like aria2c):

```
> gcm aria2c -all
...\shims\aria2c.ps1
...\shims\aria2c.exe
```

The command `scoop which aria2c` show `'aria2c' not found / not a scoop shim.`.

Reply:
The new shim system should use only `.exe` shims, and `.ps1` shims are only used for `.cmd` or bash script.

Try remove all shims except `scoop.*` and `scoop reset *`, and this should be fixed.